### ISSUE TITLE
dag_drawer not working

### ISSUE THREAD

**Issue by weiT1993** [Qiskit/qiskit-terra#3605]
### Information

- **Qiskit Terra version**: 0.8.2
- **Python version**: 3.7.5
- **Operating system**: MacOS 10.15.2

### What is the current behavior?
```dag_drawer``` raises:
```
AttributeError: 'MultiDiGraph' object has no attribute 'number_of_selfloops'
```

### Steps to reproduce the problem
```
from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
from qiskit.dagcircuit import DAGCircuit
from qiskit.converters import circuit_to_dag
from qiskit.visualization import dag_drawer

q = QuantumRegister(3, 'q')
c = ClassicalRegister(3, 'c')
circ = QuantumCircuit(q, c)
circ.h(q[0])
circ.cx(q[0], q[1])
circ.measure(q[0], c[0])
circ.rz(0.5, q[1]).c_if(c, 2)

dag = circuit_to_dag(circ)
dag_drawer(dag)
```

### What is the expected behavior?
```dag_drawer``` should draw the dag for the above circuit.

**Comment by kdk** [comment]
Hi @weiT1993 , thanks for reporting. I'm not currently able to replicate this. Can you post the full traceback along with your
```
networkx.__version__
qiskit.__version__
qiskit.__qiskit_version__
```

**Comment by ajavadia** [comment]
qiskit 0.8.2 is a bit old, i'd try updating. though i don't recall this failing at that version either.

**Comment by mtreinish** [comment]
This is an issue caused by networkx 2.4 on qiskit-terra <0.11.0 (caused by https://github.com/chebee7i/nxpd/issues/14). There are 2 ways to fix it either downgrade networkx to 2.3 or upgrade qiskit-terra to 0.11.0. I'm going to close this since it's already been fixed on the latest version of terra (by #3280), but please feel free to reopen if I'm missing something
